pangeo_forge_runner/bakery/flink.py

```diff
@@ -128,6 +128,26 @@ class FlinkOperatorBakery(Bakery):
         """,
     )
 
+    parallelism = Integer(
+        -1,
```

Comment:
Instead of special casing `-1`, I suggest we set the default to `None`, pass `allow_none=True`, and then not pass these parameters through to flink if the values are `None.

Reply:
Done in https://github.com/pangeo-forge/pangeo-forge-runner/pull/82/commits/267034e636f4ba4533158d86a1d4c34971ec60c1 👍